zhaquirks/ikea/dimmer.py

```diff
@@ -0,0 +1,107 @@
+"""Device handler for IKEA of Sweden TRADFRI wireless dimmer ICTC-G-1."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomDevice
+from zigpy.zcl.clusters.general import (
+    Basic,
+    Groups,
+    Identify,
+    LevelControl,
+    OnOff,
+    Ota,
+    PowerConfiguration,
+    PollControl,
+)
+from zigpy.zcl.clusters.lightlink import LightLink
+
+from . import IKEA
+from .. import DoublingPowerConfigurationCluster
+from ..const import (
+    ARGS,
+    CLUSTER_ID,
+    COMMAND,
+    COMMAND_MOVE,
+    DEVICE_TYPE,
+    ENDPOINT_ID,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    LEFT,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+    RIGHT,
+)
+
+ROTATED = "device_rotated"
```

Comment:
This is required for the ``device_automation_triggers`` and is also present in the ``symfonisk.py``.
Perhaps this should be moved into the ``__init.py__`` in the IKEA "folder" and then be imported in both quirks?

Reply:
either move it into `__init__.py` or you could do `from .sympfonisk import ROTATED`